docs/guides/transactions/running-background-jobs.md

```diff
@@ -100,69 +87,30 @@ class Program
 
         // Since this is not a Starcounter thread now, so we need to schedule
         // a Starcounter task so we can do our database operations.
-        await Scheduling.RunTask(() => {
-
+        await Scheduling.RunTask(async () => {
             // Querying the person.
             var person = Db.SQL<Person>("SELECT p FROM Person p WHERE p.Id = ?", id.ToString()).FirstOrDefault();
 
             // Doing some database operations.
             // Notice that we use `Db.TransactAsync` here. Read about it in a separate article.
-            Db.TransactAsync(() => person.Id += id);
-
-            // Finally returning the response.
-            Response resp = new Response() {
-                Body = "Done with object " + id
-            };
-
-            req.SendResponse(resp);
-        });
+            await Db.TransactAsync(() => person.Id += id);
+
+                // Now in continuation we need to perform some other Starcounter operations
+                // therefore we need to use `Scheduling.RunTask` again.
+                await Scheduling.RunTask(() => {
```

Comment:
@alemoi, what do you think about avoiding nested RunTask if not needed. Here it's not needed